### ISSUE TITLE
Data labels in Descriptive Overview style

### ISSUE THREAD

**Issue by MomoMoses** [johnjung/bmrcportal#71]
They are now styled as italic. I preferred the previous plain or bold (?) black type with a colon, then the data value. I think italics can be harder to read. Accessibility?
It's Ok for the selected facets since you've already read them. :)
